### PR TITLE
chore: Prepare 0.9.0

### LINKED ISSUE #2180: Release 0.9.0
I'd like to get `0.9.0` out this week so that we can start to focus on launching Kubernetes with `0.10.0`.

### LINKED ISSUE #2358: Fix broken install script
Current our [install script](https://github.com/timberio/vector/blob/master/distribution/install.sh) (sh.vector.dev) is broken attempting to add the downloaded binary into the `$PATH`. But this prompted some questions, and I need some feedback on the best behavior. For background:

1. Previously, the install script detected package managers (`dpkg`, `rpm`, `brew`, etc) and attempted to install Vector using those. Unfortunately, some of these require `sudo` privileges, `dpkg` specifically. So the script would error unless it was invoked with `sudo`, which is crazy for script downloaded from the internet. Definitely not something I want to tell users to do, even if it is our own script.
2. #1265 removed packager managers as a result.
3. So now, currently, the install script installs solely from our `*.tar.gz` archives into a `$PWD/vector` directory. This, I think, matches the `sh.rustup.rs` behavior?
4. This also has the added benefit of producing a consistent end state which is useful for step-by-step tutorials. Ex: install vector -> modify `$PWD/vector/config/vector.toml`.

So my questions are:

1. Do we all agree this is the best user experience? For example, do we want someone on say, Debian, to use this script when we have a `.deb` package that likely offers a better UX? In other words, should we drop the script all together?
2. If we want to keep the script, does it make sense for this install script to attempt to modify the `$PATH`? Are there any good examples of how to do this across multiple platforms?

### PR DESCRIPTION
**Pull request by binarylogic**
Closes #2180
Closes #2358

### CODE REVIEW

File: website/highlights/2020-03-10-dedupe-transform.md [hunk 1]
```diff
@@ -1,20 +1,42 @@
 ---
-last_modified_on: "2020-04-13"
+last_modified_on: "2020-04-16"
 $schema: "/.meta/.schemas/highlights.json"
 title: "New Dedupe Trasnform"
```
Comment (JeanMertz) [inline website/highlights/2020-03-10-dedupe-transform.md:4]:
```suggestion
title: "New Dedupe Transform"
```

Comment (JeanMertz) [reply website/highlights/2020-03-10-dedupe-transform.md:4]: We should add a lint to detect spelling mistakes for commonly used terms in Vector 🥇

File: website/highlights/2020-04-01-kubernetes-pod-metadata-transform.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ title: "New Kubernetes Pod Metdata Transform"
 description: "Easily rnrich your logs with Kubernetes metadata"
```
Comment (JeanMertz) [inline website/highlights/2020-04-01-kubernetes-pod-metadata-transform.md:5]:
```suggestion
description: "Easily enrich your logs with Kubernetes metadata"
```

File: website/highlights/2020-03-10-dedupe-transform.md (hunk 1 above)
Comment (binarylogic) [reply website/highlights/2020-03-10-dedupe-transform.md:4]: Agree, opened https://github.com/timberio/vector/issues/2357

File: website/releases/0.9.0.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
...ature","domain: transforms","transform: filter"],"title":"New Filter Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-01T00:00:00","description":"Easily rnrich your logs with Kubernetes metadata","permalink":"/highlights/2020-04-01-kubernetes-pod-metadata-transform/","prNumbers":[1888],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Kubernetes Pod Metdata Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-07T00:00:00","description":"The next iteration of our Lua transform.","permalink":"/highlights/2020-04-07-lua-transform-version-2/","prNumbers":[2126],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"Lua Transform v2","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-11T00:00:00","description":"We identified optimizations that net ~36% higher throughput","permalink":"/highlights/2020-04-11-overall-performance-increase/","prNumbers":[2295,2296],"release":"0.9.0","tags":["type: performance"],"title":"~36% Higher Throughput","type":"performance"}]}
```
Comment (LucioFranco) [inline website/releases/0.9.0.md:30]:
```suggestion
  items={[{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-23T00:00:00","description":"This brings the sink naming inline with consistent naming pattern","permalink":"/highlights/2020-03-23-rename-gcp_stackdriver_logging-sink-to-gcp_stackdriver_logs/","prNumbers":[2121],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: splunk_hec"],"title":"The `gcp_stackdriver_logging` sink has been renamed to `gcp_stackdriver_logs`","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-05T00:00:00","description":"Compression is not supported by all Elasticsearch service providers","permalink":"/highlights/2020-04-05-default-compression-to-none/","prNumbers":[2219],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: elasticsearch"],"title":"The `elasticsearch` sink `compression` option now defaults to `none`","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-09T00:00:00","description":"ACL is not always required when creating objects in GCP Cloud Storage","permalink":"/highlights/2020-04-09-make-acl-optional/","prNumbers":[2283],"release":"0.9.0","tags":["type: breaking change","domain: sinks","sink: gcp_cloud_storage"],"title":"ACL is now optional for the Google Cloud Storage sink","type":"breaking change"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-02-28T00:00:00","description":"Build Vector with select components","permalink":"/highlights/2020-02-28-custom-vector-builds/","prNumbers":[1924],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"À La Carte Custom Vector Builds","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-10T00:00:00","description":"Shed duplicate logs","permalink":"/highlights/2020-03-10-dedupe-transform/","prNumbers":[1848],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Dedupe Trasnform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-11T00:00:00","description":"Protect downstream metrics storages from runaway metrics tags","permalink":"/highlights/2020-03-11-tag-cardinality-limit-transform/","prNumbers":[1959],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: tag_cardinality_limit"],"title":"New Tag Cardinality Limit Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-03-31T00:00:00","description":"Filter and route your logs based on defined conditions","permalink":"/highlights/2020-03-31-filter-transform/","prNumbers":[2088],"release":"0.9.0","tags":["type: new feature","domain: transforms","transform: filter"],"title":"New Filter Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-01T00:00:00","description":"Easily rnrich your logs with Kubernetes metadata","permalink":"/highlights/2020-04-01-kubernetes-pod-metadata-transform/","prNumbers":[1888],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"New Kubernetes Pod Metdata Transform","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-07T00:00:00","description":"The next iteration of our Lua transform.","permalink":"/highlights/2020-04-07-lua-transform-version-2/","prNumbers":[2126],"release":"0.9.0","tags":["type: new feature","domain: sources","source: vector"],"title":"Lua Transform v2","type":"new feature"},{"authorGithub":"https://github.com/binarylogic","dateString":"2020-04-11T00:00:00","description":"We identified optimizations that net ~36% higher throughput","permalink":"/highlights/2020-04-11-overall-performance-increase/","prNumbers":[2295,2296],"release":"0.9.0","tags":["type: performance"],"title":"~36% Higher Throughput","type":"performance"}]}
```

It says stackdrive on the preview but it should be stackdriver?

Comment (binarylogic) [reply website/releases/0.9.0.md]: Thanks, fixed.

**Pull request merged**